src/geo/projection/globe_util.js

```diff
@@ -44,7 +44,7 @@ const GLOBE_LOW_ZOOM_TILE_AABBS = [
     new Aabb([0, 0, GLOBE_MIN], [GLOBE_MAX, GLOBE_MAX, GLOBE_MAX])  // x=1, y=1
 ];
 
-const GLOBE_VERTEX_GRID_SIZE_LOD_TABLE = [64, 54, 36, 32, 30];
+const GLOBE_VERTEX_GRID_SIZE_LOD_TABLE = [64, 64, 64, 64, 32];
```

Comment:
What kind of camera parameters do I need to use to see the LOD in action?

Reply:
@mpulkki-mapbox I fixed a bug that I had accidentally introduced when calculating the tile center latitude used in the LOD calculation. The top image is what came before taking latitude into account. The lower image takes latitude into account:
![without_latitude_change](https://user-images.githubusercontent.com/67509069/157860005-0a409071-9570-4764-b49a-4045a5644267.png)
![with_latitude_change](https://user-images.githubusercontent.com/67509069/157860022-69b8111c-b4f6-413f-96a7-ddb89df8a8ad.png)
Notice how tiles closer to the pole have fewer subdivisions. To take things even further we could subdivide tiles rather based on their distance from map center. It's perhaps a good thing to do this in a separate PR in future. cc @karimnaaji